cmd/cluster_etcd_backup.go

```diff
@@ -0,0 +1,37 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+	"github.com/xetys/hetzner-kube/pkg/clustermanager"
+	"github.com/xetys/hetzner-kube/pkg/hetzner"
+)
+
+var backupCmd = &cobra.Command{
+	Use:     "backup",
+	Short:   "creates a backup of the etcd cluster. If no name is provided, a current datetime string is used",
+	PreRunE: validateClusterInArgumentExists,
+	Run: func(cmd *cobra.Command, args []string) {
+		snapshotName, _ := cmd.Flags().GetString("snapshot-name")
+		etcdManager := getEtcdManager(cmd, args)
+
+		err := etcdManager.CreateSnapshot(snapshotName)
+
+		if err != nil {
+			fmt.Println(err)
```

Comment:
Also return error code if we have error and we trap it, otherwise return error to cobra. RunE to allow the cobra library to return formatted message and error code, eg:

```go
var backupCmd = &cobra.Command{
	// ...
	RunE: func(cmd *cobra.Command, args []string) error {
		snapshotName, _ := cmd.Flags().GetString("snapshot-name")
		etcdManager := getEtcdManager(cmd, args)

 		return etcdManager.CreateSnapshot(snapshotName)
	}
```